rpc/item.c

```diff
@@ -566,15 +623,31 @@ M0_INTERNAL bool m0_rpc_item_xid_check(struct m0_rpc_item *item,
 	if (!rpc_item_needs_xid(item))
 		return true;
 
-	M0_LOG(M0_DEBUG, "item: "ITEM_FMT" session=%p xid=%"PRIu64
-	       " s_xid=%"PRIu64, ITEM_ARG(item), sess, xid, sess->s_xid);
+	M0_LOG(M0_DEBUG, "item: "ITEM_FMT" session=%p osr_xid=%"PRIu64" "
+	       "osr_session_xid_min=%"PRIu64" s_xid=%"PRIu64,
+	       ITEM_ARG(item), sess, item->ri_header.osr_session_xid_min,
+	       xid, sess->s_xid);
```

Comment:
The order of arguments "item->ri_header.osr_session_xid_min, xid" are wrong.

Reply:
Fixed.